jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -232,6 +241,45 @@ public void testDockerDaemon_simpleWithIncompatibleJava11()
     }
   }
 
+  @Test
+  public void testDockerDaemon_simple_deprecatedExtraDirectory()
+      throws DigestException, IOException, InterruptedException {
+    String targetImage = "localhost:6000/simpleimage:gradle" + System.nanoTime();
```

Comment:
Seems like you could reduce some code duplication between these 3 tests if you move all of this into `assertExtraDirectoryDeprecation` and take the build.gradle name as a parameter?